### DISCUSSION TITLE
Inconsistent behavior with cut_with_point_loop() on similar meshes

### DISCUSSION THREAD

**Discussion by Linus-Foley** [marcomusy/vedo#1189] [General]
Hi, just wondering what it is that determines which side is kept when using `cut_with_point_loop()`.

I'm using `cut_with_point_loop()` on a few hundred meshes, and I'm having an issue where for most cases, setting `invert=True` is giving me the result that I want. But there's one mesh where I need to set `invert=False` to get the desired result. The normals and winding appear to be correct, is there anything else that could affect this? Visually my examples are all very similar, so I'm looking for any potential non-visual causes.

**Comment by marcomusy** [comment]
Hi, this might be depending on 2 things: the triangle orientation, and the orientation of the points sequece that defines the loop.
You can change the first with `.reverse()` ..but at the end of the day it's the same effect as invert=True.

**Reply by Linus-Foley** [reply]
Thanks Marco! Interestingly, both calling `.reverse()` and reversing the points sequence produce no change in the result. Yet, changing `invert` does. Quite the pickle... if you have any other thoughts, please let me know.

**Comment by marcomusy** [comment]
Hi, I guess my only way to help you would be if you can share two small example meshes on which I can reproduce the issue ....
